pkg/manager/manager_tasks_test.go

```diff
@@ -0,0 +1,59 @@
+package manager
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestExcludeFiles(t *testing.T) {
+
+	filenames := []string{
+		"/stash/videos/filename.mp4",
```

Comment:
For completeness, can we add windows network paths as well? `\\\\network-share\\path\\etc`

Reply:
Added a few.
A lot of `\` escapes needed ....

Reply:
Windows network paths are with double backslash `\\network-share\etc`.
```
\\\\\\\\network\\\\videos\\\\filename  windows network.mp4
```
translates to 
```
\\\\network\\videos\\filename  windows network.mp4
```
It should be similar to the `c:\\` ones - ie:
```
\\\\network\\videos\\filename  windows network.mp4
```
which would translate to 
```
\\network\videos\filename  windows network.mp4
```
which is the correct form.

Reply:
i mistook your `\\\\network-share\\path\\etc` as the non escaped path instead of already escaped , now fixed